pkg/bootstrap/docker/up.go

```diff
@@ -857,7 +869,7 @@ func (c *ClientStartConfig) InstallLogging(out io.Writer) error {
 	if len(publicMaster) == 0 {
 		publicMaster = c.ServerIP
 	}
-	return c.OpenShiftHelper().InstallLogging(f, publicMaster, openshift.LoggingHost(c.RoutingSuffix, c.ServerIP), c.Image, c.ImageVersion)
+	return c.OpenShiftHelper().InstallLogging(f, publicMaster, openshift.LoggingHost(c.RoutingSuffix, c.ServerIP), defaultLoggingImagePrefix, c.ImageVersion)
```

Comment:
@smarterclayton decided to hard-code this instead of exposing a new flag since we expect this to go away in 3.5. This does expose us to the case where the user is using a RHEL-based `oc cluster up` but wants to deploy Origin with it... should that be added as a validation step to make it impossible?